include/zephyr/bluetooth/services/ias.h

```diff
@@ -21,6 +21,42 @@
 extern "C" {
 #endif
 
+#include <stdint.h>
+#include <bluetooth/gatt.h>
```

Comment:
There's no need to include `gatt.h`

Reply:
Done (but I think I still need it in ias_internal.h)